config/zones/CH.yaml

```diff
@@ -73,11 +73,30 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 113.07156762843732
     unknown:
-      _comment:
-        'Source ENTSOE, Swiss government and Electricity Maps: https://github.com/electricitymaps/electricitymaps-contrib/pull/2898
-        and https://docs.google.com/spreadsheets/d/1FLHQ6e9Es08BIqX654BM3SEb_fuAk4k4O-6cmRXyw_E/'
-      source: 52% hydro storage, 48% unknown thermal
-      value: 288
+      - datetime: '2017-01-01'
+        _comment: 'Assumed make up of the unknown production is detailed here: https://docs.google.com/spreadsheets/d/1FLHQ6e9Es08BIqX654BM3SEb_fuAk4k4O-6cmRXyw_E'
+        source: SFOE, Electricity Maps
```

Comment:
To help figure out merge conflict resolution for #6001: I _think_ this indicates that the linked spreadsheet combines data and/or discussion from SFOE with Electricity Maps' own calculations.

Also as context: the spreadsheet is an [existing `source` entry for `hydro`](https://github.com/electricitymaps/electricitymaps-contrib/blob/e53c5a41c2dfc72effd8afc50e8e356cf8049b12/config/zones/CH.yaml#L301) -- seeming to reaffirm that it's derived calculations, now with input from SFOE in addition.

Based on that, to resolve #6001 I think I'd lean towards adding a single top-level `source` entry to the zone YAML file, named `SFOE, Electricity Maps`.  I did notice that a few other zones use semi-colons to indicate that multiple sources were referenced to gather metrics.. this case seems potentially different, in that it's one document that has been produced by two groups in collaboration.

Please let me know if that's roughly on-track!

Reply:
> I think this indicates that the linked spreadsheet combines data and/or discussion from SFOE with Electricity Maps' own calculations.

Correct

> Also as context: the spreadsheet is an [existing source entry for hydro](https://github.com/electricitymaps/electricitymaps-contrib/blob/e53c5a41c2dfc72effd8afc50e8e356cf8049b12/config/zones/CH.yaml#L301) -- seeming to reaffirm that it's derived calculations, now with input from SFOE in addition.

Yes, this will be updated in the coming days as data is reprocessed with the new unknown emission factor. We will be recomputing the emission factor for hydro discharge as well. 

>  I did notice that a few other zones use semi-colons to indicate that multiple sources were referenced to gather metrics..

There is no single convention here, this is only used as information for users to get a sense of where the data is coming from. 

> I think I'd lean towards adding a single top-level source entry to the zone YAML

Not sure exactly what you have in mind, but be aware that we might have multiple sources for one zone (ex: one source for each emission factor)